cls/SourceControl/Git/Settings.cls

```diff
@@ -341,4 +348,36 @@ Method OnAfterConfigure() As %Boolean
     }
 }
 
+Method ConfigureBinPath(ByRef path As %String) As %Boolean
+{
+    if (path = "") { return 1 }
+    // Sometimes path is quoted 
+    set path = $replace(path,"""","")
+    if (##class(%File).DirectoryExists(path)) {
+        set gitFile = $case($zversion(1),2:"git.exe",3:"git",0:"git")
```

Comment:
One more nit, can we say 2:"git.exe",:"git") to avoid <ILLEGAL VALUE> for whatever 1 is?

Reply:
There's also a $$$isWINDOWS macro somewhere that would be cleaner.